sled-agent/src/services.rs

```diff
@@ -2852,17 +2853,68 @@ impl ServiceManager {
                         SwitchService::Tfport { pkt_source, asic } => {
                             info!(self.inner.log, "Setting up tfport service");
                             let mut tfport_config =
-                                PropertyGroupBuilder::new("config")
+                                PropertyGroupBuilder::new("config");
+
+                            tfport_config = tfport_config
+                                .add_property(
+                                    "host",
+                                    "astring",
+                                    &format!("[{}]", Ipv6Addr::LOCALHOST),
+                                )
+                                .add_property(
+                                    "port",
+                                    "astring",
+                                    &format!("{}", DENDRITE_PORT),
+                                );
+
+                            if let Some(i) = info {
+                                tfport_config = tfport_config
+                                    .add_property(
```

Comment:
Rather than repeating this same code 5 (6?) times, maybe add a helper function to do it?  That will also help avoid any future errors if we decide to add/change a property.

Reply:
Yeah. For sure. This piece of code is generally super repetitive, but I'll make this change for sled properties.